src/types/README.md

```diff
@@ -30,9 +30,20 @@ Checks for native type `number`
 
 Checks for native type `object`
 
+## Phone
+
+Checks for valid phone numbers:
+
+* `phone`: Default, valid with or without separators
+* `phone:international`: Checks value for format `+XX-XXXX-XXXX`
```

Comment:
Is this documented somewhere? I feel like this can't be true.. there simply aren't enough digits.

The international form of a US number, for example, is `+1-XXX-XXX-XXXX` which I believe can also be dialed in long-form as `+001-XXX-XXX-XXXX`